settings/base.py

```diff
@@ -49,7 +61,7 @@ def path(*args):
 STATIC_URL = '/static/'
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '0D8AE44F-5714-40EF-9AC8-4AC6EB556161'
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
My reasoning for not providing a default for this and `SUPERTOKEN` were for security. If the defaults for these two values are not overridden in a deployed site it would be quite bad. But you're right that doing it this way means that even if you have `SECRET_KEY` in `local.py` it will still raise an exception if it's not in the env. Hmmm... I'll think on it some more.